tests/network/libs/bgp.py

```
@@ -270,6 +330,30 @@ def wait_for_bgp_connection_established(node_names: list) -> None:
+        frr_pod: The external FRR pod to query.
+        expected_neighbors: Number of neighbors expected to have active EVPN sessions.
+    """
+    output = frr_pod.execute(
+        command=shlex.split('vtysh -c "show bgp l2vpn evpn summary json"'),
+        container="frr",
+    )
+    summary = json.loads(output)
+    peers = summary.get("peers", {})
+    active_count = sum(1 for peer_info in peers.values() if peer_info.get("pfxRcd", 0) > 0)
+    if active_count < expected_neighbors:
+        raise RuntimeError(f"EVPN not fully established: {active_count}/{expected_neighbors} neighbors active")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)